yt_dlp/extractor/roosterteeth.py

```diff
@@ -130,6 +129,30 @@ class RoosterTeethIE(RoosterTeethBaseIE):
             'release_date': '20141016',
         },
         'params': {'skip_download': True},
+    }, {
+        'url': 'https://roosterteeth.com/watch/rwby-bonus-21',
+        'info_dict': {
+            'id': '33',
```

Comment:
are we sure this `id` value is unique for the bonus features?

Reply:
the id field is unique for all bonus features, but not for for "normal" episodes and bonus features combined
example: `hardcore-tabletop-the-greenroom` and `cow-chop-gaming-2016-worst-hand-simulator-e-v-e-r`

Reply:
Hm, perhaps we should append a `-bonus` suffix to bonus feature IDs then? We'll need to use `smuggle_url` for that, and it could be messy.

Or we could use the `episode_id` value as the `id` instead? We'd still need to extract the old ID as `_old_archive_ids` that way. But I guess this doesn't really matter much if the site is shutting down